src/uct/tcp/tcp_ep.c

```diff
@@ -591,6 +591,75 @@ static ucs_status_t uct_tcp_ep_keepalive_enable(uct_tcp_ep_t *ep)
 #endif /* UCT_TCP_EP_KEEPALIVE */
 }
 
+static void uct_tcp_iface_check_rp_filter(uct_tcp_iface_t *iface)
+{
+    const struct sockaddr* iface_saddr =
+            (struct sockaddr*)&iface->config.ifaddr;
+    char rp_filter_path[MAXPATHLEN];
+    long rp_filter;
+    ucs_status_t status;
+
+    ucs_assert(ucs_sockaddr_is_known_af(iface_saddr));
+
+    if (iface_saddr->sa_family != AF_INET) {
```

Comment:
can't we support ipv6 as well?

Reply:
there is no `rp_filter` configuration for ipv6